projects/1.20-fabric/assets/mythic-charms/mythic_charms/lang/zh_cn.json

```
@@ -0,0 +1,191 @@
+
+	"item.mythic_charms.unbreakable_charm_of_feathered_grace": "坚固的优雅之羽护符",
+	"item.mythic_charms.unbreakable_charm_of_blazing_embrace": "坚固的炽热拥抱护符",
+	"item.mythic_charms.unbreakable_charm_of_gaze_serenity": "坚固的宁静眺望护符",
+	"item.mythic_charms.unbreakable_charm_of_earths_order": "坚固的地球意志护符",
+	"item.mythic_charms.unbreakable_charm_of_botanic_blessing": "坚固的植物祝福护符",
+	"item.mythic_charms.unbreakable_charm_of_fleeting_strides": "坚固的轻快步伐护符",
+	"item.mythic_charms.unbreakable_charm_of_nights_guardian": "坚固的暗夜守护者护符",
+	"item.mythic_charms.unbreakable_charm_of_high_bounds": "坚固的高界护符",
+	"item.mythic_charms.unbreakable_charm_of_drowned_freedom": "坚固的自由水漫护符",
+	"item.mythic_charms.unbreakable_charm_of_weightless_flow": "坚固的失重之流护符",
+	"item.mythic_charms.unbreakable_charm_of_collectors_gift": "坚固的收藏家之礼护符",
+	"item.mythic_charms.unbreakable_charm_of_climbers_path": "坚固的攀山之径护符",
+	"item.mythic_charms.unbreakable_charm_of_natures_call": "坚固的自然召唤护符",
+	"item.mythic_charms.unbreakable_charm_of_barters_pact": "坚固的贸易之约护符",
+	"item.mythic_charms.unbreakable_charm_of_battle_fury": "坚固的狂战士护符",
+	"item.mythic_charms.unbreakable_charm_of_echoing_wrath": "坚固的狂怒回响护符",
+	"item.mythic_charms.unbreakable_charm_of_enchanted_whispers": "坚固的魔法低语护符",
+	"item.mythic_charms.unbreakable_charm_of_arrow_dance": "坚固的箭舞护符",
+	"item.mythic_charms.unbreakable_charm_of_mountains_strength": "坚固的山峦之力护符",
+	"item.mythic_charms.unbreakable_charm_of_safe_territory": "坚固的安全领土护符",
+	"item.mythic_charms.unbreakable_charm_of_quiet_presence": "坚固的阒然无声护符",
+
+
+	"item.mythic_charms.feathered_grace.tooltip": "为佩戴者在坠落半秒后且没有潜行的情况下给予两秒缓降效果",
+
+	"item.mythic_charms.blazing_embrace.tooltip": "永久给予佩戴者耐火效果，但只有当佩戴者没有着火时，持续时间才会延长",
+
+	"item.mythic_charms.earths_order.tooltip": "使佩戴者能够仅用徒手破坏任何可破坏的方块",
+
+	"item.mythic_charms.gaze_serenity.tooltip": "使佩戴者能够直视末影人且不会触怒末影人，并且在战斗中不会触发随机传送",
+
+	"item.mythic_charms.botanic_blessing.tooltip": "加速佩戴者周围两格半径内农作物的生长",
+
+	"item.mythic_charms.fleeting_strides.tooltip": "提高佩戴者跳跃和奔跑时的速度，但会额外消耗饱食度",
+
+	"item.mythic_charms.nights_guardian.tooltip": "防止幻翼生成（仅针对佩戴者），即使周围有幻翼，它们也会无视佩戴者",
+
+	"item.mythic_charms.high_bounds.tooltip": "提高佩戴者的跳跃高度和安全落地高度，使他们在一格内不会受到伤害，但会额外消耗饱食度",
+
+	"item.mythic_charms.drowned_freedom.tooltip": "赋予佩戴者在水下像在水面上一样移动的能力，但会额外消耗饱食度",
+
+	"item.mythic_charms.weightless_flow.tooltip": "锁定佩戴者周围的重力使其可以站在空中或跃上天空，同时无法潜行",
+
+	"item.mythic_charms.collectors_gift.tooltip": "使佩戴者可以收集自身两格半径内的物品",
+
+	"item.mythic_charms.climbers_path.tooltip": "使佩戴者可以额外跨越一格的高度以轻松越过障碍物，但是会额外消耗饱食度",
+
+	"item.mythic_charms.natures_call.tooltip": "在佩戴者不潜行的情况下，将动物吸引到佩戴者周围，就像佩戴者拿着动物最喜欢的食物一样",
+
+	"item.mythic_charms.barters_pact.tooltip": "佩戴者打开箱子时有70%的几率不会引发猪灵的攻击",
+
+	"item.mythic_charms.battle_fury.tooltip": "当佩戴者的生命值下降时，其攻击伤害会增加；当他们只剩下半颗心时，伤害输出会达到正常伤害输出的三倍",
+
+	"item.mythic_charms.echoing_wrath.tooltip": "当佩戴者受到伤害时，将该伤害的50%反弹给附近的小怪，最多造成3颗心的伤害；佩戴者受到的伤害增加25%",
+
+	"item.mythic_charms.enchanted_whispers.tooltip": "每次佩戴者与附魔台互动时都会随机化一个1级的附魔效果",

```

```suggestion
	"item.mythic_charms.enchanted_whispers.tooltip": "每次佩戴者与附魔台互动时都会重新随机魔咒，但魔咒等级会下降1级",
```
这里需要到游戏内验证是魔咒的等级还是附魔时候右下角数字的最大值
注意原文语义

---

这里说实话我没有搞明白具体作用，因为游戏内实际操作发现是否佩戴该护符似乎不会影响到附魔结果【很奇妙】